### ISSUE TITLE
Questions about the dataset

### ISSUE THREAD

**Issue by zhangleino1** [BehnamZeinali/IMUNet#1]
Thank you for your generosity in sharing the code.
But in the ridi model, your validation set and test set are the same. Is this a problem?

**Comment by Xyanpian** [comment]
> Thank you for your generosity in sharing the code. But in the ridi model, your validation set and test set are the same. Is this a problem?

Do you know how to use the oxiod dataset? Do all scenarios need to be used? The readoxiod.py in the code does not have a train.txt file corresponding to the large scale

**Comment by Xyanpian** [comment]
> > Thank you for your generosity in sharing the code. But in the ridi model, your validation set and test set are the same. Is this a problem?
> 
> Do you know how to use the oxiod dataset? Do all scenarios need to be used? The readoxiod.py in the code does not have a train.txt file corresponding to the large scale



> Thank you for your generosity in sharing the code. But in the ridi model, your validation set and test set are the same. Is this a problem?

Do you know how to use the oxiod dataset? Do all scenarios need to be used? The readoxiod.py in the code does not have a train.txt file corresponding to the large scale

**Comment by zhangleino1** [comment]
Although the author is open source code, but there are some reservations, I feel that may be deliberately set the bar

**Comment by zhangleino1** [comment]
你可以自己设置
